ClientCore/ProgramConstants.cs

```diff
@@ -42,6 +43,19 @@ public static class ProgramConstants
         public const string SPAWNER_SETTINGS = "spawn.ini";
         public const string SAVED_GAME_SPAWN_INI = "Saved Games/spawnSG.ini";
 
+        /// <summary>
+        /// The language code that corresponds to the language the hardcoded client strings are in.
+        /// </summary>
+        public const string HARDCODED_LOCALE_CODE = "en";
+
+        /// <summary>
+        /// Used to denote <see cref="Environment.NewLine"/> in the INI files.
+        /// </summary>
+        /// <remarks>
+        /// Historically Westwood used '@' for this purpose, so we keep it for compatibility.
+        /// </remarks>
+        public const string INI_NEWLINE_PATTERN = "@";
```

Comment:
How to support displaying email addresses?

Reply:
I am sticking to client inherited westwood's norms, you can't display an email address in INI files either which is fine IMO.

Reply:
If very needed, can add support for it via something like `\@`.

Reply:
Replied at https://github.com/CnCNet/xna-cncnet-client/pull/409#issuecomment-1356167513